canva.com.website.json

```diff
@@ -4,27 +4,37 @@
    "serviceId":"website",
    "serviceName":"Canva Websites",
    "syncPubKeyDomain":"domainconnect.canva.com",
-   "version": 2,
-   "logoUrl": "https://static.canva.com/static/images/canva_logo_100x100@2x.png",
+   "version":3,
+   "logoUrl":"https://static.canva.com/static/images/canva_logo.svg",
    "description":"Enables a domain to work with Canva",
-   "variableDescription":"verify is the verification value for the site",
-   "records":[  
-      {  
+   "variableDescription":"verify-name and verify-value are used for the name and value of a verification record",
+   "records":[
+      {
+         "groupId":"hosting",
          "type":"A",
          "host":"@",
          "pointsTo":"103.169.142.0",
          "ttl":300
       },
-      {  
+      {
+         "groupId":"hosting",
          "type":"A",
          "host":"www",
          "pointsTo":"103.169.142.0",
          "ttl":300
       },
       {
+         "groupId":"verificationTxt",
          "type":"TXT",
-         "host": "_canva-domain-verify",
-         "data": "%verify%",
+         "host":"%verify-name%",
```

Comment:
Are we able to specify `@` for this variable?

e.g. `https://web-connect.dnsprovider.com/v2/domainTemplates/providers/canva/services/template/apply?domain=example.com&verify-name=@&verify-value=123`

Reply:
This way it won't work. The specification does not cover for such chaining of variables.
But you can use %fqdn% directly in the template

Reply:
Ah ok. Just to be sure I know what you mean, is it true that the query string can chain variables, like so?
`?domain=example.com&host=foo&var1=abc&var2=123`

But you're saying that we can't use `@` as a value here. Is that correct?

If that's the case, we can hardcode `"host":"@"` into our template here 👍 But for the CNAME, we'd still like to use `%verify-name%` (since it will always be defined, and will never be `@`).

Some examples of how we plan to use this template are below:

| Intended verification record | Query String |
---
| CNAME `_verify.example.com` -> `123.canva.com` | `domain=example.com&host=foo&verify-name=_verify&verify-value=123.verify.canva.com&groupId=hosting,verificationCname` |
| TXT `@` -> `verify=123` | `domain=example.com&host=foo&verify-value=verify%3D123&groupId=hosting,verificationTxt` ** |

**I'm assuming we encode `=` to `%3D` if passing it in as a variable

Reply:
Yes, this will work. On a side note, putting TXT record on apex has some downsides so you may want to reconsider:
https://www.ietf.org/archive/id/draft-ietf-dnsop-domain-verification-techniques-01.html#name-txt-based

BTW: you can test all of that using an online editor https://pdnsadmin.revproxy.short-lived.de/dc/free/templateedit
Just a remark - this implementation actually allows for @ in a variable - but this may be an exception.

Reply:
Thanks @pawel-kow, I forgot about the online editor and it's perfect!

Since this template version does include breaking changes, I've opted to create a new template in https://github.com/Domain-Connect/Templates/pull/541